tunelinux/iscsi.go

```diff
@@ -389,6 +392,7 @@ func SetIscsiRecommendations() (err error) {
 
 // ConfigureIscsi verifies and install necessary iSCSI packages if not present. It also makes sure if service is running.
 func ConfigureIscsi() (err error) {
+	global := false
```

Comment:
make global as argument here too..i will have to change all callers(across dcs/cloudvol, csi-driver etc) after this to pass this boolean as true(as most of the times we need host level setting)

Reply:
ConfigureIscsi is being called from dockervolume component. There are multiple levels involved in the calling of ConfigureIscsi and for host level settings nimbletune --set category -global will take care right?